packages/rancher-logging/overlay/templates/loggings/eks/logging.yaml

```diff
@@ -24,4 +24,8 @@ spec:
     image:
       repository: {{ template "system_default_registry" . }}{{ .Values.images.fluentd.repository }}
       tag: {{ .Values.images.fluentd.tag }}
+    configReloaderImage:
```

Comment:
I don't think it needs to happen for this pr, but it seems like the fluentd section is the same in all of these logging yamls and might be a good candidate for a helper template